CHANGELOG-core.md

```diff
@@ -25,6 +26,11 @@ This release contains a number of features aimed at improving the node's reliabi
 - `ETH_FINALITY_DEPTH` specifies how deep protection should be against re-orgs. The default is 50. It only applies if BulletproofTxManager is enabled. It is not recommended to change this setting.
 - `EthHeadTrackerHistoryDepth` specifies how many heads the head tracker should keep in the database. The default is 100. It is not recommended to change this setting.
 
+### Breaking changes
+
+- `admin withdraw` command has been removed. This was only ever useful to withdraw LINK if the Oracle contract was owned by the Chainlink node address. It is recommended to have the Oracle owner be a different address than the Chainlink node address.
+- Fixed `txs create` to send the amount in Eth not in Wei (as per the documentation)
+
```

Comment:
```suggestion
### Changed

- Fixed `txs create` to send the amount in Eth not in Wei (as per the documentation)

### Removed

- `admin withdraw` command has been removed. This was only ever useful to withdraw LINK if the Oracle contract was owned by the Chainlink node address. It is recommended to have the Oracle owner be a different address than the Chainlink node address.

```

We're trying to follow the [KeepAChangelog](https://keepachangelog.com/en/1.0.0/) format, which supports 6 categories(Added, Changed, Deprecated, Removed, Fixed, and Security). I hope that sticking to these categories will make it easier for readers to parse out what they care about. (I missed the `Env var changes section` last time, but would ideally like to roll that into this categorization as well.

Reply:
Didn't know that, nice :+1: